pages/biosero_cherrypick.vue

```diff
@@ -86,15 +86,10 @@ export default {
     handleResponse(response) {
       let message, type
       if (response.success) {
-        if (response.errors) {
-          message = response.errors.join(', ')
-          type = 'warning'
-        } else {
-          message = response.response
-          type = 'success'
-        }
+        message = response.response
```

Comment:
response.response? is that right?

Reply:
Yeah it's ugly but kept the same name as I couldn't think of a better name during the refactor. :P